src/DynamoCoreWpf/ViewModels/Watch3D/AttachedProperties.cs

```diff
@@ -0,0 +1,56 @@
+﻿using System.Windows;
+
+using HelixToolkit.Wpf.SharpDX;
+
+namespace Dynamo.Wpf.ViewModels.Watch3D
+{
+    public static class AttachedProperties
```

Comment:
Again, needs better docs so other developers can understand what this code is for.